app/api/utility.py

```diff
@@ -55,16 +65,29 @@ def create_query(
     """
     subject_level_filters = ""
 
-    if isinstance(age, tuple):
-        if age[0] is not None:
-            subject_level_filters += "\n" + f"FILTER (?{AGE.var} >= {age[0]})."
-        if age[1] is not None:
-            subject_level_filters += "\n" + f"FILTER (?{AGE.var} <= {age[1]})."
+    if age[0] is not None:
```

Comment:
this is good, I agree with doing it. But I think in the future it'll be faster to do this in a separate PR because it is unrelated to the diagnosis / control_subject context here and those are already a bit complex.

Reply:
Noted, will do!